administrator/language/en-GB/com_guidedtours.ini

```
@@ -4,6 +4,12 @@
 ; Note : All ini files need to be saved as UTF-8
 
 COM_GUIDEDTOURS="Guided Tours"
+COM_GUIDEDTOURS_CONFIG_DELAYED_TIME_DESCRIPTION="The amount of time (in seconds) a tour is delayed after being cancelled by the user and until it is shown again (only when a tour is set to start automatically).<br />For instance, enter 3600 for 1 hour, 86400 for 24 hours, 604800 for 1 week."
+COM_GUIDEDTOURS_CONFIG_DELAYED_TIME_LABEL="Auto Start Time Delay"
+COM_GUIDEDTOURS_CONFIG_USER_TOURAUTOSTART_DESCRIPTION="Turn on or off the auto starting functionality of tours"
```

```suggestion
COM_GUIDEDTOURS_CONFIG_USER_TOURAUTOSTART_DESCRIPTION="Turn on or off the auto starting functionality of tours."
```

A sentence should end with a .